index.html

```diff
@@ -106,7 +106,7 @@ <h1>
     <section>
         <div class="works" id="works-id">
             <h1>My Recent Works</h1>
-            <article class="card-work">
+            <article class="card-work ">
```

Comment:
- [ ] Good job so far, :clap: , however, please ensure that your design matches the given design by adjusting the alignment of your fonts in `.headline p` , the content of your `.social-media` and `.about-myself p` classes, as part of the project requirement that says: `You must replicate the design as much as possible (e.g., font, colors, images, text, margins)` using this [template](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?type=design&node-id=1-468&mode=design&t=5Ue5vUAb207bYWOz-0). As it is right now, your background image is stretching, matching your design to the given design requirements enhances the chances of creating a successful and well-rounded design solution that meets the needs of users, complies with regulations, and performs effectively and efficiently. :slightly_smiling_face: 

The example below shows the difference between yours and the accepted one: :arrow_down: 

 <table>
    <tr>
      <td>Yours </td>
      <td>Figma</td>
    </tr>
    <tr>
      <td>

 
 
![StudentH](https://github.com/malodev1234/Portfolio_desktop_version/assets/43528594/18a4304a-3956-487b-b904-06db25ce32a0)


 
</td>
      <td>

![Figma Home](https://github.com/malodev1234/Portfolio_desktop_version/assets/43528594/c9395a90-6022-4534-a94c-9711f985e4c8)

</td>
    </tr>
<tr>
      <td>

 
 
![Student about](https://github.com/malodev1234/Portfolio_desktop_version/assets/43528594/7b86b802-e5bb-4e3f-9eea-b0fe20c6916c)


 
</td>
      <td>

![Figma about](https://github.com/malodev1234/Portfolio_desktop_version/assets/43528594/8e739cdc-94c9-41a1-a8bd-722a0b032868)

</td>
    </tr>
  </table>